### ISSUE TITLE
[Hammer] smaller grid sizes

### ISSUE THREAD

**Issue by Konclan** [ChaosInitiative/Portal-2-Community-Edition#450]
An option to have smaller grid sizes like 0.5, 0.25, and 0.1

**Comment by L89David** [comment]
But why...

**Comment by Luke18033** [comment]
For precise entity positioning? You can already put entities in sub-unit positions using `Ctrl+M`, so this would just make it more convenient.

**Comment by L89David** [comment]
I can't think of a situation where you would use subunits

**Comment by Luke18033** [comment]
Though brushes should maybe still be limited to 1 unit thick at the minimum, since I think thinner brushes can cause errors.

**Comment by L89David** [comment]
That makes more sense, but the thing that I thought is that when would you really want to make so precise movement and having such a small grid for it.

**Comment by Mystical-Ace** [comment]
I've experiecned scenarios where I need to move a brush or a prop on a sub-unit scale. 
And thickness on a sub-unit scale can cause errors, I've had the unfortunate error of brushes disappearing when I open the map if they happen to be 0.1u and placed in some way the game doesn't like it.

**Comment by Konclan** [comment]
I mean you can already make broken geometry in hammer without sub-unit scale grid. Especially with shapes like cylinders you already have vertices going onto 0.5 and further

**Comment by L89David** [comment]
Yeah but you don't really move objects in subunits that much to have more grid sizes. You can just use what Luke said, or alt dragging (wich is not precise of course).
